sdk/core/azure-core/src/http/url.cpp

```diff
@@ -219,7 +219,10 @@ std::string Url::GetUrlWithoutQuery(bool relative) const
   {
     if (!relative)
     {
-      url += "/";
+      if (m_encodedPath.empty() || m_encodedPath[0] != '/')
```

Comment:
This condition `m_encodedPath.empty()` will never be true since it is in an if block that says `!m_encodedPath.empty()`, above.

Reply:
#5210